chapters/pre-recorded-stt/speaker-diarization.mdx

```diff
@@ -25,17 +25,20 @@ Breaking changes may still be introduced to this API, but an advanced notice wil
 We're looking for feedback to improve this feature, [share yours here](https://gladiaio.typeform.com/feedback?typeform-source=app.gladia.io).
 </Note>
 
-For improved diarization handling edge cases and challenging audio, you can enable enhanced diarization by using the `diarization_enhanced` parameter.
+For improved diarization handling edge cases and challenging audio, you can enable enhanced diarization by using the `enhanced` parameter in the `diarization_config` object.
```

Comment:
Precise some kind of a trade-off or you will get spammed by "if it's better, why not enabled by default?".

Example trade-off : Bêta, Slower

Reply:
Updated the block note on top